python/pyspark/sql/dataframe.py

```diff
@@ -868,6 +875,17 @@ def fillna(self, value, subset=None):
 
             return DataFrame(self._jdf.na().fill(value, self._jseq(subset)), self.sql_ctx)
 
+    def cov(self, col1, col2):
+        """
+        Calculate the covariance for the given columns, specified by their names.
+        alias for ``stat.cov()``.
+
+        :param col1: The name of the first column
+        :param col2: The name of the second column
+        :return: the covariance of the columns
```

Comment:
does ```:return:``` work in python doc? Can you try compiling it?

Also say something about the return type, maybe "covariance of the two columns as a double value"